From 0f19909f0b368dd3f98fdd64f7813cd1875f6fb2 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 19 Dec 2023 14:04:58 +0100
Subject: [PATCH 1066/1141] dt-bindings: media: st,stm32mp25-vdec: add
 'power-domains' property

STM32 VDEC may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 VDEC.

Fix dtbs_check warning:
 Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I8900051378426af7bc888bbc2d0fa2f7e357b614
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347870
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 Documentation/devicetree/bindings/media/st,stm32mp25-vdec.yaml | 3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/devicetree/bindings/media/st,stm32mp25-vdec.yaml
+++ b/Documentation/devicetree/bindings/media/st,stm32mp25-vdec.yaml
@@ -36,6 +36,9 @@ properties:
   clock-names:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
 required:
   - compatible
   - reg
